nano/rpc_test/rpc.cpp

```diff
@@ -6397,10 +6397,10 @@ TEST (rpc, active_difficulty)
 	request.put ("action", "active_difficulty");
 	node->active.multipliers_cb.push_front (1.5);
 	node->active.multipliers_cb.push_front (4.2);
-	std::mutex mutex;
-	std::unique_lock<std::mutex> lock (mutex);
+	std::unique_lock<std::mutex> lock (node->active.mutex);
```

Comment:
Best lock before pushing the values. Also fixes an intermittent issue where a `1` is placed in between them and the test fails.

Reply:
Thanks, moved the lock